common/src/stack/ansible/Makefile

```diff
@@ -0,0 +1,22 @@
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+
+ROLLROOT = ../../../..
+include $(STACKBUILD)/etc/CCRules.mk
+
+build:
+
+install::
+	# Install the Ansible config
+	mkdir -p $(ROOT)/etc/ansible/
+	$(INSTALL) -m0644 etc/ansible.cfg $(ROOT)/etc/ansible/ansible.cfg
```

Comment:
In reviewing this PR, I'm going back and forth with myself on making our `ansible.cfg` packaged-and-placed-by-default in /etc.  If I'm not mistaken, that means that if someone edits /etc/ansible.cfg, maybe because they don't want to use our ansible+python at all, that there's packaging issues.  I wonder if it should be in /opt/stack/etc, and symlinked in node xml?

Reply:
We were writing out the file in the previous node XML, but I like the idea of making it a symlink to something in /opt/stack. In fact, I took that approach when I redid the Apache stuff, so I should have done that here.

BTW: Thanks for the useful code review comments (cough cough... Ryan... cough cough)